The integration tests don't pass with Python 3.8

The `test_wait_for_jobs_timeout` integration test produces an error with Python 3.8:

```
    async def wait_for_jobs(connection: aiopg.Connection, socket_timeout: float):
        try:
            await asyncio.wait_for(connection.notifies.get(), timeout=socket_timeout)
>       except asyncio.futures.TimeoutError:
E       AttributeError: module 'asyncio.futures' has no attribute 'TimeoutError'
```

`TimeoutError` is no longer in `asyncio.futures`. And the docs say to use `asyncio.TimeoutError` anyway, which should work with at least Python 3.5, 3.6, 3.7 and 3.8.

PR to come.

--- a/procrastinate/aiopg_connector.py
+++ b/procrastinate/aiopg_connector.py
@@ -61,7 +61,7 @@ def make_dynamic_query(query: str, **identifiers: str) -> str:
 async def wait_for_jobs(connection: aiopg.Connection, socket_timeout: float):
     try:
         await asyncio.wait_for(connection.notifies.get(), timeout=socket_timeout)
-    except asyncio.futures.TimeoutError:
+    except asyncio.TimeoutError:
         pass
 
 

